src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Pages/EmailConfirmation.razor.cs

```diff
@@ -62,7 +55,7 @@ await HttpClient.PostAsJsonAsync("Auth/SendConfirmationEmail", new()
 
             EmailConfirmationMessageType = BitMessageBarType.Success;
 
-            EmailConfirmationMessage = "The confirmation link has been re-sent.";
+            EmailConfirmationMessage = "The confirmation email has been re-sent.";
```

Comment:
plz change the text to `The confirmation link has been re-sent to your email address.`